### ISSUE TITLE
TI link cable emulation

### ISSUE THREAD

**Issue by scj643** [KermMartian/ArTICL#8]
Is it possible to make this act as if it was a TI 83+ link cable? I can't make it myself but I do have the hardware to do it.

**Comment by KermMartian** [comment]
Yes, ArTICL can be used to build a link cable. In fact, geekboy1011 of Cemetech has been working on exactly that: https://www.cemetech.net/news.php?id=668

**Comment by scj643** [comment]
Is their source code for that though?

**Comment by geekbozu** [comment]
So I made something specific to the arduino Leonardo. I have yet to make something that works with any other hardware. What platform will you end up using? Its available as is here https://bitbucket.org/Geekboy1011/silverlinkclone

Sadly I have just started actually using github. But if you want I can throw it up here for PRs and stuff if you plan on modifying it.

**Comment by scj643** [comment]
I'm using an Arduino Uno.

**Comment by geekbozu** [comment]
For what I have i have not made it work with V-USB It is possible I
believe, i forget if V-USB supports bulk endpoints, but  regardless it
would need some work. Right now im leveraging the atmega32uX usb controller
to make the usb side of things really easy. Sadly I do not have time to
work on it. Would you care to fork it and work on it?

**Comment by scj643** [comment]
I don't have the know how to do so

**Comment by KermMartian** [comment]
@geekbozu should keep working on https://bitbucket.org/Geekboy1011/silverlinkclone; because it's not core functionality of ArTICL, I'm going to close this for now.
